Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7049.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using NUnit.Framework;
+using Xamarin.UITest;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve (AllMembers=true)]
+	[Issue (IssueTracker.Github, 7049, "Null reference exception on some Android devices - Xamarin.Forms.Platform.Android.PageRenderer.Xamarin.Forms.Platform.Android.IOrderedTraversalController.UpdateTraversalOrder", PlatformAffected.Android)]
+	public class Issue7049 : TestContentPage
+	{
+		private bool _flag = true;
+
+		protected override void Init()
+		{
+
+			var button = new Button { Text = "Continue" };
+			button.Clicked += (_, __) =>
+			{
+				var view = _flag ?
+					(View)new Entry { AutomationId = "View2", Text = "Press 1 time to crash" } :
+					new Label { AutomationId = "View3", Text = "I'm shown, bug fixed!" };
+				_flag ^= true;
+				(Content as StackLayout).Children[1] = view;
+			};
+			Content = new StackLayout
+			{
+				Children =
+				{
+					button,
+					new Label { AutomationId = "View1", Text = "Press 2 times to crash" }
+				}
+			};
+		}
+
+#if UITEST
```

Comment:
Since this is specific to Android, could you also add `&& __ANDROID__`?

Reply:
Done